### PR TITLE
fix RunCommand calls to use string array

### PR DESCRIPTION
**Pull request by krwq**

### CODE REVIEW

File: src/Microsoft.DotNet.Tools.Run/RunCommand.cs [hunk 1]
```diff
@@ -90,7 +90,18 @@ private int RunExecutable()
             var tempDir = Path.Combine(_context.ProjectDirectory, "bin", ".dotnetrun", Guid.NewGuid().ToString("N"));
 
             // Compile to that directory
-            var result = Command.Create($"dotnet-build", $"--output \"{tempDir}\" --temp-output \"{tempDir}\" --framework \"{_context.TargetFramework}\" --configuration \"{Configuration}\" {_context.ProjectFile.ProjectDirectory}")
+            var result = Command.Create($"dotnet-build", new string[]
+				{
```
Comment (krwq) [inline src/Microsoft.DotNet.Tools.Run/RunCommand.cs]: formatting broke

File: src/Microsoft.DotNet.Tools.Run/RunCommand.cs [hunk 2]
```diff
@@ -132,7 +143,7 @@ private int RunExecutable()
                 }
             }
 
-            result = Command.Create(outputName, string.Join(" ", _args))
+            result = Command.Create(outputName, _args.ToArray())
```
Comment (brthor) [inline src/Microsoft.DotNet.Tools.Run/RunCommand.cs:146]: Nice, just want to double check that this hasn't already been quoted?

Comment (krwq) [inline src/Microsoft.DotNet.Tools.Run/RunCommand.cs:146]: right, looking

Comment (krwq) [inline src/Microsoft.DotNet.Tools.Run/RunCommand.cs:146]: just --debug and filename don't see any quotes

**Pull request merged**